src/main/kotlin/racingcar/view/ResultView.kt

```diff
@@ -22,11 +22,17 @@ object ResultView {
         return winnerNameView
     }
 
-    fun showRecordsAsDash(results: MoveResults) {
-        results.result.map { result ->
+    fun showAllRacingRecords(results: List<MoveResults>) {
+        results.forEach { eachResult ->
+            showEachRecord(eachResult)
+            println()
+        }
+    }
+
+    private fun showEachRecord(moveResults: MoveResults) {
+        moveResults.data.map { result ->
             showResultView(result.carName, getDashView(result.moveDistance))
         }
```

Comment:
```suggestion
    fun showAllRacingRecords(results: List<MoveResults>) {
        results.forEach(::showEachRecord)
    }

    private fun showEachRecord(moveResults: MoveResults) {
        moveResults.data.map { result ->
            showResultView(result.carName, getDashView(result.moveDistance))
        }
        println()
```

[method reference](https://proandroiddev.com/kotlin-lambda-vs-method-reference-fdbd175f6845)를 활용하면 arguments로 전달되는 값이 수정되지 않았음을 보장할 수 있다는 장점이 있어요. 이번 기회에 사용법을 익혀보시면 어떨까요?

Reply:
리뷰 반영했습니다 :)